modules/rubiconAnalyticsAdapter.js

```diff
@@ -157,6 +159,15 @@ function sendMessage(auctionId, bidWonId) {
         adUnit.bids = [];
       }
 
+      // Add site and zone id if not there and if we found a rubicon bidder
+      if ((!adUnit.siteId || !adUnit.zoneId) && rubiconAliases.indexOf(bid.bidder) >= 0) {
```

Comment:
This is nitpicky, but instead of greater or equal to 0, I would do not equal to -1
 `rubiconAliases.indexOf(bid.bidder) !== -1`

Reply:
Yeah I like that better